backfill/2020-01-23-sync-ping/append_to_prod.sql

```diff
@@ -0,0 +1,40 @@
+WITH ids AS (
+  SELECT
+    DATE(submission_timestamp) AS dt,
+    document_id
+  FROM
+    `moz-fx-data-shared-prod.telemetry_stable.crash_v4`
+  WHERE
+    DATE(submission_timestamp)
+    BETWEEN '2019-12-04'
+    AND '2020-01-09'
+  UNION ALL
+  SELECT
+    DATE(submission_timestamp) AS dt,
+    document_id
+  FROM
+    `moz-fx-data-backfill-2.telemetry_stable.crash_v4`
```

Comment:
Shouldn't be:
```suggestion
    `moz-fx-data-backfill-4.telemetry_stable.crash_v4`
```
here and below?

Reply:
Looks like this whole script was just copied from the crash backfill and not edited yet